<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_data</class>
 <widget class="QDialog" name="dialog_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1a1a1a;
color: #ffffff;
font: bold 9pt &quot;Lato&quot;;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#242424</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffbe00;
font: 14pt &quot;Lato&quot;;</string>
    </property>
    <property name="text">
     <string>MODIFY DATA</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>450</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#242424;
font: 9pt &quot;Lato Black&quot;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#2d2d2d;
border:none;</string>
    </property>
    <property name="text">
     <string>CANCEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifyButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#2d2d2d;
border:none;</string>
    </property>
    <property name="text">
     <string>MODIFY</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>450</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	background:transparent;
	border: 3px solid #000000;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>434</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QCheckBox" name="yd_cb_ve">
       <property name="styleSheet">
        <string notr="true">QCheckBox{
	color: #7f7f7f;
}

QCheckBox:Checked{
	color: #ffbe00;
}</string>
       </property>
       <property name="text">
        <string>V EXTENSION</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="yd_cb_ce">
       <property name="styleSheet">
        <string notr="true">QCheckBox{
	color: #7f7f7f;
}

QCheckBox:Checked{
	color: #ffbe00;
}</string>
       </property>
       <property name="text">
        <string>CLAZZIQUAI</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="yd_cb_gf">
       <property name="styleSheet">
        <string notr="true">QCheckBox{
	color: #7f7f7f;
}

QCheckBox:Checked{
	color: #ffbe00;
}</string>
       </property>
       <property name="text">
        <string>GIRLS' FRONTLINE</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="yd_cb_bs">
       <property name="styleSheet">
        <string notr="true">QCheckBox{
	color: #7f7f7f;
}

QCheckBox:Checked{
	color: #ffbe00;
}</string>
       </property>
       <property name="text">
        <string>BLACK SQUARE</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="yd_cb_es">
       <property name="styleSheet">
        <string notr="true">QCheckBox{
	color: #7f7f7f;
}

QCheckBox:Checked{
	color: #ffbe00;
}</string>
       </property>
       <property name="text">
        <string>EMOTIONAL S.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="yd_cb_t3">
       <property name="styleSheet">
        <string notr="true">QCheckBox{
	color: #7f7f7f;
}

QCheckBox:Checked{
	color: #ffbe00;
}</string>
       </property>
       <property name="text">
        <string>TECHNIKA 3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="yd_cb_t1">
       <property name="styleSheet">
        <string notr="true">QCheckBox{
	color: #7f7f7f;
}

QCheckBox:Checked{
	color: #ffbe00;
}</string>
       </property>
       <property name="text">
        <string>TECHNIKA 1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="yd_cb_t2">
       <property name="styleSheet">
        <string notr="true">QCheckBox{
	color: #7f7f7f;
}

QCheckBox:Checked{
	color: #ffbe00;
}</string>
       </property>
       <property name="text">
        <string>TECHNIKA 2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="yd_cb_cy">
       <property name="styleSheet">
        <string notr="true">QCheckBox{
	color: #7f7f7f;
}

QCheckBox:Checked{
	color: #ffbe00;
}</string>
       </property>
       <property name="text">
        <string>CYTUS</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="yd_cb_tr">
       <property name="styleSheet">
        <string notr="true">QCheckBox{
	color: #7f7f7f;
}

QCheckBox:Checked{
	color: #ffbe00;
}</string>
       </property>
       <property name="text">
        <string>TRILOGY</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="yd_cb_gc">
       <property name="styleSheet">
        <string notr="true">QCheckBox{
	color: #7f7f7f;
}

QCheckBox:Checked{
	color: #ffbe00;
}</string>
       </property>
       <property name="text">
        <string>GROOVE COASTER</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="yd_cb_dm">
       <property name="styleSheet">
        <string notr="true">QCheckBox{
	color: #7f7f7f;
}

QCheckBox:Checked{
	color: #ffbe00;
}</string>
       </property>
       <property name="text">
        <string>DEEMO</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="yd_cb_chu">
       <property name="styleSheet">
        <string notr="true">QCheckBox{
	color: #7f7f7f;
}

QCheckBox:Checked{
	color: #ffbe00;
}</string>
       </property>
       <property name="text">
        <string>CHUNITHM</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>450</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border: none;
font: 10pt &quot;Lato&quot;;</string>
    </property>
    <property name="text">
     <string>Select Your Own DLCs.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <zorder>frame_3</zorder>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
